core/web/chains_controller.go

```diff
@@ -101,6 +108,12 @@ func (cc *chainsController[I, C, R]) Create(c *gin.Context) {
 		return
 	}
 
+	chainj, err := json.Marshal(chain)
+	if err != nil {
+		cc.lggr.Errorf("Unable to marshal chain to json", "err", err)
+	}
+	cc.auditLogger.Audit(audit.ChainAdded, map[string]interface{}{"chain": chainj})
```

Comment:
If an error is returned, there is no guarantee about the contents of `chainj`. It might be empty, but could also contain partial or non-sensical data. It could be set nil explicity:
```suggestion
		cc.lggr.Errorf("Unable to marshal chain to json", "err", err)
		chainj = nil
	}
	cc.auditLogger.Audit(audit.ChainAdded, map[string]interface{}{"chain": chainj})
```
Or we could log something different that notes the error? :shrug: